tensorflow/contrib/cmake/BUILD

```diff
@@ -0,0 +1,46 @@
+licenses(["notice"])  # Apache 2.0
+
+package(default_visibility = ["//visibility:public"])
+
+load("//tensorflow:tensorflow.bzl", "py_test")
+
+py_test(
```

Comment:
You also needs the deps field here, and that will need to have the BUILD target for the platform.test library.

Reply:
Should be `//tensorflow/python:platform_test` ?

Reply:
Correct.